internal/provider-config/config.go

```diff
@@ -423,6 +423,10 @@ func determineIncusDir() (string, error) {
 	incusSocketPaths := []string{
 		"/var/lib/incus/unix.socket",
 		"/var/lib/incus/unix.socket.user",
+		"/run/incus/unix.socket",
+		"/run/incus/unix.socket.user",
+		"/usr/lib/systemd/system/incus.socket",
+		"/usr/lib/systemd/system/incus-user.socket",
```

Comment:
Those two shouldn't be included, they're not socket but systemd unit files.

Reply:
Ooops, sorry for that (I must've copied the wrong path from running `systemctl status incus.socket`). Also thank you for catching it.

Reply:
I also adjusted the commit message since I rechecked it and it should work for both Fedora (on which I tested) and OpenSUSE (Arch already uses the same path as Debian/Ubuntu).